app/components/Header/DropdownContainer/index.js

```diff
@@ -0,0 +1,139 @@
+// @flow
+
+import React, { Component, Children, createRef } from 'react';
+import type { Node } from 'react';
+import { Flipped } from 'react-flip-toolkit';
+import styled, { keyframes } from 'styled-components';
+import FadeContents from './FadeContents';
+import styles from '../Header.css';
+import { any } from 'prop-types';
+
+type Props = {
+  children: Node,
+  animatingOut: boolean,
+  direction: 'left' | 'right',
+};
+
+const getFirstDropdownSectionHeight = (el) => {
+  if (
+    !el ||
+    !el.querySelector ||
+    !el.querySelector('*[data-first-dropdown-section]')
+  )
+    return 0;
+  return el.querySelector('*[data-first-dropdown-section]').offsetHeight;
```

Comment:
This should be a `try catch`, i think that looks a bit more clear
```js
try {
  return el.querySelector('*[data-first-dropdown-section]').offsetHeight
} catch {
  return 0
}
```

Reply:
Ye it made things a lot clearer 👍🏼